_test/test_setup.go

```diff
@@ -177,12 +179,9 @@ func spinUpTestContainer(t *testing.T, rchan chan<- *dockertest.Resource, pool *
 
 	t.Logf("Chain ID %s spun up in container %s from %s", c.ChainID(), resource.Container.Name, resource.Container.Config.Image)
 
-	// retry polling the container until status doesn't error
-	//if err = pool.Retry(c.StatusErr); err != nil {
-	//	return fmt.Errorf("could not connect to container at %s: %s", c.RPCAddr, err)
-	//}
-
-	// TODO maybe this works?
+	// we used to poll here until the container is running without status errors but,
+	// we no longer expose the status error on the relayer.Chain struct.
+	// this sleep statement seems to
```

Comment:
Seems to what?

Reply:
Ahh don't know how that one happened, i'll add the rest back in. 

The sleep statement works fine and it was what was already being used for awhile now but I figured some clarification on why it was there was nice.